src/js/shared/reducers/shot-generator.js

```diff
@@ -796,6 +801,25 @@ const sceneObjectsReducer = (state = {}, action) => {
         }
         return
 
+      case 'UPDATE_CHARACTER_IK_SKELETON':
+          draft[action.payload.id].skeleton = draft[action.payload.id].skeleton || {}
+          for(let bone of action.payload.skeleton)
+          {
+            let rotation = bone.rotation;
+            let position = bone.position;
+            draft[action.payload.id].skeleton[bone.name] = { 
+              rotation : {x: rotation.x, y : rotation.y, z: rotation.z},
+              position : {x: position.x, y : position.y, z: position.z }
```

Comment:
Is `position` used?

Reply:
Nope, it only changes the rotation of bones.